common/ini-file.cpp

```diff
@@ -125,7 +126,7 @@ bool INIFile::loadFromStream(SeekableReadStream &stream) {
 			section.comment = comment;
 			comment.clear();
 
-			assert(isValidName(section.name));
+			assert(_8bitMode || isValidName(section.name));
```

Comment:
It would be better to extend the functionality of isValidName(), instead of short-circuiting its usage all over the place